src/themes/__styles__.css

```diff
@@ -0,0 +1,222 @@
+/*
+        Ffau - A blocky-based editor for teaching HTML, CSS and Javascript.
+
+				Developed by Pal Kerecsenyi, Geza Kerecsenyi and Oli Plant.
+				Full details are avaliable at the Github repo: https://github.com/codeddraig/ffau
+				Ffau editor will not work without its libraries. The best way to get all
+					off this data at once is to grab the latest release version from the
+					Github repo.
+				Ffau is open source software. This means you can re-mix, share and use
+					it however you want, including for commercial purposes. However, you
+					MUST provide attribution to the original authors if you do this.
+				However, Ffau is provided with NO WARRANTY whatsoever, and by using this
+					software, you agree to the terms of the MIT License.
+
+				Copyright (c) 2017-19 The CodeDdraig Organisation
+
+				THIS IS VERSION 1.0.0
+*/
+
+.cog-icon {
+    color: rgba(0, 0, 0, 0.3);
+    transition: color 120ms ease-in 95ms, transform 300ms ease-out, -webkit-transform 300ms ease-out;
+}
+
+.settings-button.closed:hover > .cog-icon {
+    color: rgba(0, 0, 0, 0.6);
+}
+
+.settings-button.open:hover > .cog-icon, .settings-button.closed:hover > .cog-icon {
+    -webkit-transform: rotate(280deg);
+    transform: rotate(280deg);
+}
+
+.settings-button.opening > .cog-icon, .settings-button.open > .cog-icon {
+    color: rgba(255, 255, 255, 0.6);
+}
+
+.settings-button.closing > .cog-icon {
+    color: rgba(128, 128, 128, 0.6);
+}
+
+.settings-header {
+    display: inline;
+    margin-left: 10px;
+    top: 3px;
+    font-size: 16px;
+    color: rgba(255, 255, 255, 0.8);
+    text-align: center;
+    position: relative;
+}
+
+.settings-button {
+    position: absolute;
+    left: -31px;
+    z-index: 251;
+    top: -26px;
+    background-color: rgba(117, 117, 117, 0.4);
+    padding: 2px 10px 2px 5px;
+    border-radius: 10px 0 0 10px;
+    cursor: pointer;
+    transform: rotate(0);
+    transition: background-color 90ms linear,
+    padding-right 90ms cubic-bezier(0.21, 0.9, 0.88, 0.72),
```

Comment:
Please indent these lines and put a newline after `transition:`. See https://ibb.co/CmXW21m (bad) vs https://ibb.co/Sv9yLJx (good)

Reply:
I've fixed this now